src/components/AsanaCallback.tsx

```diff
@@ -0,0 +1,44 @@
+
+import React, { useEffect } from 'react';
+import { RouteComponentProps, navigate } from '@reach/router';
+import { ASANA_REDIRECT_URI } from '../lib/asana';
+import { message } from 'antd';
+import CenterSpin from './CenterSpin';
+import { useMount } from 'react-use';
```

Comment:
TIL about this pretty cool package! Thanks!

Reply:
It is fantastic. I only recently starting using it and I love all of it.